src/transformers/trainer.py

```diff
@@ -2831,6 +2831,12 @@ def save_model(self, output_dir: Optional[str] = None, _internal_call: bool = Fa
                 state_dict = self.accelerator.get_state_dict(self.model)
                 if self.args.should_save:
                     self._save(output_dir, state_dict=state_dict)
+            else:
+                if not version.parse(accelerate_version) > version.parse("0.24.1"):
+                    logger.warning(
+                        "I can't save FSDP normally because the version of accelerate is low!"
+                        " Please increase the version of accelerate to 0.24.5 or higher!"
+                    )
```

Comment:
```suggestion
            elif version.parse(accelerate_version) <= version.parse("0.24.1") and ("FULL_STATE_DICT" in str(self.accelerator.state.fsdp_plugin.state_dict_type)):
                    raise ValueError(
                        "To save full state dict in transformers format, the minimum version of 🤗 Accelerate required is 0.24.2 or higher. Current version: {version.parse(accelerate_version)}."
                    )
```

Reply:
@pacman100 Thanks for taking me up on my suggestion, it looks good to me! 
I think we should add a relevant warning in training_args.py or trainer.py to let people know about this before they start training, what do you think?